src/snowflake/cli/_plugins/nativeapp/codegen/setup/native_app_setup_processor.py

```diff
@@ -40,6 +46,16 @@
 DEFAULT_TIMEOUT = 30
 DRIVER_PATH = Path(__file__).parent / "setup_driver.py.source"
 
+log = logging.getLogger(__name__)
+
+
+def safe_set(d: dict, *keys: str, **kwargs) -> None:
```

Comment:
I could use a docstring here to understand the intention

Reply:
fair, done